chapters/es/slides/chapter2_04_models-rules.md

```diff
@@ -94,24 +94,24 @@ token en el documento.
 
 ```python
 matcher = Matcher(nlp.vocab)
-matcher.add("DOG", None, [{"LOWER": "golden"}, {"LOWER": "retriever"}])
-doc = nlp("I have a Golden Retriever")
+matcher.add("PERRO", None, [{"LOWER": "golden"}, {"LOWER": "retriever"}])
+doc = nlp("Tengo un Golden Retriever")
 
 for match_id, start, end in matcher(doc):
     span = doc[start:end]
-    print("Matched span:", span.text)
-    # Obtén el token raíz del span y el token raíz central (head)
-    print("Root token:", span.root.text)
-    print("Root head token:", span.root.head.text)
+    print("span encontrado:", span.text)
+    # Obtén el token raíz del span y el token raíz cabeza (head)
+    print("Token raíz:", span.root.text)
+    print("Token raíz cabeza:", span.root.head.text)
     # Obtén el token anterior y su POS tag
-    print("Previous token:", doc[start - 1].text, doc[start - 1].pos_)
+    print("Token anterior:", doc[start - 1].text, doc[start - 1].pos_)
 ```
 
 ```out
-Matched span: Golden Retriever
-Root token: Retriever
-Root head token: have
-Previous token: a DET
+span encontrado: Golden Retriever
+Token raíz: Golden
```

Comment:
We probably need a better example here (had the same problem in German). The head of "Golden Retriever" assigned by the Spanish model is actually "Golden" and not "Retriever", which makes the whole point here kinda confusing and unintuitive. For German, I used a different dog breed with an adjective, "deutscher Kurzhaardackel" (German short hair dachshund 😅) so the head of the span correcctly came back as the noun that governs the phrase.

Reply:
Solved it! I changed it to the actual Spanish name for a Golden Retriever -> Labrador Dorado. Now it takes "Labrador/Retriever" as the token root.